examples/tutorials/basics/symbols.py

```diff
@@ -0,0 +1,94 @@
+"""
+Plotting symbols
+================
+
+The :meth:`pygmt.Figure.plot` method can plot symbols via the ``style``,
+``size``, and ``symbol`` parameters.
+"""
+
+# %%
+import pygmt
+
+# Set up five sample data points as lists for the x and y values
+x = [-4, -2, 0, 2, 4]
+y = [0] * len(x)
+
+
+# %%
+# Use the ``style`` parameter of the :meth:`pygmt.Figure.plot` method to plot all data
+# points with the same symbol and size. Via the ``fill`` parameter the symbols can be
+# filled with a color or pattern, and ``pen`` can add an outline (pass an argument in
```

Comment:
```suggestion
# points with the same symbol and size. The symbols can be
# filled with a color or pattern via the ``fill`` parameter, and 
# ``pen`` can add an outline (pass an argument in
```

Reply:
@michaelgrund thanks for already looking at this PR. However, to be more precise regarding single- and multi-parameter symbols I made larger changes to documentation and moved this general part on the `fill` and `pen` parameter to the introduction at the top.

Reply:
Actually I made more changes. Now there are examples for `pen` and `fill`, and I moved this part of the docs there.